lib/core/basebandApiManager.cpp

```diff
@@ -1,39 +1,45 @@
 #include "basebandApiManager.hpp"
 
-#include <iostream>
-#include <sstream>
-
 #include "basebandReadoutManager.hpp"
 #include "kotekanLogging.hpp"
 
+#include <iostream>
+#include <sstream>
+
 
-basebandReadoutManager& basebandApiManager::basebandReadoutRegistry::operator[]( const uint32_t& key ) {
+basebandReadoutManager& basebandApiManager::basebandReadoutRegistry::
+operator[](const uint32_t& key) {
     std::lock_guard<std::mutex> lock(map_lock);
     return readout_map[key];
 }
 
-basebandApiManager::basebandReadoutRegistry::iterator basebandApiManager::basebandReadoutRegistry::begin() noexcept {
+basebandApiManager::basebandReadoutRegistry::iterator
+basebandApiManager::basebandReadoutRegistry::begin() noexcept {
     std::lock_guard<std::mutex> lock(map_lock);
     return readout_map.begin();
 }
 
-basebandApiManager::basebandReadoutRegistry::iterator basebandApiManager::basebandReadoutRegistry::end() noexcept {
+basebandApiManager::basebandReadoutRegistry::iterator
+basebandApiManager::basebandReadoutRegistry::end() noexcept {
     std::lock_guard<std::mutex> lock(map_lock);
     return readout_map.end();
 }
 
-void to_json(json &j, const basebandDumpStatus& d) {
-    j = json{{"file_name", d.request.file_name},
-             {"total", d.bytes_total},
-             {"remaining", d.bytes_remaining}};
+void to_json(json& j, const basebandDumpStatus& d) {
+    j = json {{"file_name", d.request.file_name},
```

Comment:
This is an interesting effect with initializer lists...  Seems to go against the standard of function calls not having a space, but also super confusing for an autoformater.